src/styles/components/Overlay.scss

```diff
@@ -1,10 +1,41 @@
 .Overlay {
-  opacity: 0;
   transition: opacity 3500ms cubic-bezier(0.79, 0.1, 0.56, 0.93);
   pointer-events: none;
-}
 
-.Overlay--active {
-  opacity: 1;
-  pointer-events: auto;
+  &--active {
+    pointer-events: auto;
+  }
+
+  &__navigation {
+    @include media("sm-down") {
+      position: absolute;
+      bottom: 0;
+      left: 0;
+    }
+  }
+
+  &__social-media-list {
+    a {
+      color: color("white");
+    }
+  }
```

Comment:
Any reason this couldn't be a class?

Reply:
I was trying to use `'color-white'` as a class for the `<List>` component I'm using for the social media links, and I think it's not affecting these links because the util class `.link` that has  `color: color("black")` is taking precedence.